packages/block-editor/src/components/block-list/style.scss

```diff
@@ -224,6 +224,10 @@
 		}
 	}
 
+	&.is-outline-mode.is-selected.is-hovered {
```

Comment:
@youknowriad This is simpler. Blocks use their inherent cursor when hovered _and_ selected.

Reply:
yes, this works for me. Another approach would have been a `:not(.is-selected)`  in the other selector.